### PR TITLE
Tic tac toe

### PR DESCRIPTION
**Pull request by mviruet2**

### CODE REVIEW

File: ticTacToe.py [hunk 1]
```diff
@@ -1,23 +1,25 @@
...
     
-    # TO DO #################################################################
-    # Write code in this function that checks the tic-tac-toe board          #
-    # to determine if the player stored in variable 'player' currently      #
-    # has a winning position on the board.                                  #
-    # This function should return True if the player specified in           #
-    # variable 'player' has won. The function should return False           #
-    # if the player in the variable 'player' has not won.                   #
-    #########################################################################
+return ((board['top-L'] == player and board['top-M'] == player and board['top-R'] == player) or
```
Comment (halfaro1) [inline ticTacToe.py]: @mviruet2 remember that indentation matters in Python, so you may need to indent this. Do you understand how these lines of code work?

File: ticTacToe.py [hunk 2]
```diff
@@ -1,23 +1,25 @@
+Layout of the board
```
Comment (halfaro1) [inline ticTacToe.py]: @mviruet2 this isn't valid Python code, so you'll probably get an error from your Python IDLE when you run this. Do you? Let's get together and talk about this if it doesn't make sense.

File: ticTacToe.py (hunk 1 above)
Comment (mviruet2) [inline ticTacToe.py]: @halfaro1 I'm having a little trouble understanding how the lines work. Every time I tried to fix it says error.

File: ticTacToe.py [hunk 3]
```diff
@@ -1,23 +1,24 @@
+def printBoard(board):{'top-L': ' ', 'top-M': ' ', 'top-R': ' ',
+                    'mid-L': ' ', 'mid-M': ' ', 'mid-R': ' ',
+                    'low-L': ' ', 'low-M': ' ', 'low-R': ' '}
 def printBoard(board):
```
Comment (halfaro1) [inline ticTacToe.py]: @mviruet2 there might be an issue here with the duplicate definition of the `printBoard` function

**Comment by halfaro1** [comment]
@mviruet2 did you edit the `.travis.yml` file? it is required for the testing to work

File: ticTacToe.py [hunk 4]
```diff
@@ -1,23 +1,21 @@
...
+    print('Checking if ' + player + ' is a winner...')    
+    if board['top-L'] == player and board['mid-L'] == player and board['low-L'] == player: return True
+    if board['top-M'] == player and board['mid-M'] == player and board['low-M'] == player: return True
+    if board['top-R'] == player and board['mid-R'] == player and board['low-R'] == player: return True
+    if board['top-L'] == player and board['top-M'] == player and board['top-R'] == player: return True
+    if board['mid-L'] == player and board['mid-M'] == player and board['mid-R'] == player: return True
+    if board['low-L'] == player and board['low-M'] == player and board['low-R'] == player: return True
+    if board['top-L'] == player and board['mid-M'] == player and board['low-R'] == player: return True
+    if board['top-R'] == player and board['mid-M'] == player and board['low-L'] == player: return True
+
```
Comment (halfaro1) [inline ticTacToe.py]: @mviruet2 how about when there is no winner? In those cases, you should return `False`.

**Comment by halfaro1** [comment]
Fantastic work :+1:
I'm marking this pull request as closed, and labeling it as `graded` while I finish grading these projects. When I'm done grading all the projects, and your grade should be on Blackboard, I'll change the tag to `grade updated on blackboard`

**Pull request closed without merging**